.storybook/main.ts

```diff
@@ -1,7 +1,7 @@
 import { StorybookConfig } from "@storybook/nextjs";
 
 const storybookConfig: StorybookConfig = {
-  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
```

Comment:
removed because we don't have any .mdx file stories and storybook v7 shows a warning if we include this config without them